ColyberCopter/src/VirtualPilot.cpp

```diff
@@ -57,6 +58,8 @@ bool VirtualPilot::setFlightMode(FlightModeTypes flightModeType)
     executePrepareAndLeaveMethods(currentFlightMode, flightModeToSet);
     
     currentFlightMode = flightModeToSet;
+
+    Instance::debMes.showMessage(currentFlightMode->getName());
```

Comment:
Add message that flight mode has changed besides the name of changed flight mode